bench/utils.py

```diff
@@ -300,14 +300,19 @@ def setup_app(app):
 		setup_app(app)
 
 
-def exec_cmd(cmd, cwd='.'):
+def exec_cmd(cmd, cwd='.', fail_on_non_zero_return=False):
 	import shlex
 	print("{0}$ {1}{2}".format(color.silver, cmd, color.nc))
 	cwd_info = "cd {0} && ".format(cwd) if cwd != "." else ""
 	cmd_log = "{0}{1}".format(cwd_info, cmd)
 	logger.debug(cmd_log)
 	cmd = shlex.split(cmd)
-	return_code = subprocess.call(cmd, cwd=cwd, universal_newlines=True)
+
+	if fail_on_non_zero_return:
+		return_code = subprocess.check_call(cmd, cwd=cwd, universal_newlines=True)
```

Comment:
On failure, will this display the `stderr` by the command or end with a Python traceback?